src/Tasklib.php

```diff
@@ -3,6 +3,8 @@
 
 trait Tasklib
```

Comment:
From chat:

Tasklib is for classes (outside Robo) that want to be able to use $this->taskFoo(…) without extending Tasks.